src/translations/en.json

```diff
@@ -754,6 +755,7 @@
   "SearchPage.schemaDescription": "Showing search results",
   "SearchPage.schemaForSearch": "a query",
   "SearchPage.schemaTitle": "Search results for {searchTitle} | {marketplaceName}",
+  "SeatsFilter.labelSelected":"Seats - {seatCount}",
```

Comment:
This probably needs a bit of discussion with copy-text department. :)

We currently have 3 different ways to mark which filters are active. This would be 4th => I suggest that we'll stick to existing: `"Seats: {seatCount}"`.
![Screenshot 2024-12-03 at 14 31 39](https://github.com/user-attachments/assets/08d7d96c-4a6a-426c-a52b-cca185760c78)

Reply:
This translation is used on the map page and it's specified in the design:
<img width="713" alt="Screenshot 2024-12-03 at 14 48 28" src="https://github.com/user-attachments/assets/05bc270f-71a9-4b1c-a1a6-404e4b6b4a61">

The grid page i.e. plain filter currently uses `FilterComponent.seatsLabel` like this:

```
    const labelForPlain = hasInitialValues ? (
      <span className={css.labelPlainSelected}>
        {label}: {initialSeats}
      </span>
    ) : (
      <span className={css.labelPlain}>{label}</span>
    );
```
but I can add a separate label translation for the plain version as well.

Reply:
That probably makes sense. 

I think that our Integer range should actually behave differently than it currently does:
![Screenshot 2024-12-03 at 15 24 20](https://github.com/user-attachments/assets/57b4810e-8907-451d-9934-54e1691ed3cb)

I.e. it should look like this: `Gears: 18 - 24` since random range "18 - 24" does not specify which listing field filter it is + the range should fit on that button label concept.

I guess it could be achieved with just something like this: 

```jsx
  label={intl.formatMessage({ id: 'IntegerRangeFilter.labelSelectedPopup' }, {minValue, maxValue})}
  //   "IntegerRangeFilter.labelSelectedPopup": "{label}: {minValue} - {maxValue}",
```

Reply:
Added a new key `SeatsFilter.labelSelectedPlain` for the grid page version and renamed the popup key to `SeatsFilter.labelSelectedPopup` for clarity

Reply:
Changed both versions so they use a single key, `SeatsFilter.labelSelected`, which has the colon option.